README.md

```diff
@@ -54,3 +54,190 @@ For a simple, gentler, introduction to Lagom, have a look at the [Hello World gi
 - Getting help
 
 If you have any troubles and need help, feel free to ask in the [Gitter channel](https://gitter.im/lagom/lagom)
+
+## Running: Kubernetes
+
+This project uses [Lightbend's Platform Tooling](https://s3-us-west-2.amazonaws.com/rp-tooling-temp-docs/home.html) to
+simplify deployment to [Kubernetes](https://kubernetes.io/).
+
+### Setup
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+* [Minikube](https://github.com/kubernetes/minikube) v0.25.0 or later (Verify with `minikube version`)
+* [Helm](https://github.com/kubernetes/helm)
+* [reactive-cli](https://github.com/lightbend/reactive-cli) 0.9.0 or later (Verify with `rp version`)
```

Comment:
For the reactive-cli setup, it's better to point people to https://developer.lightbend.com/docs/reactive-platform-tooling/latest/cli-installation.html#install-the-cli instead. 

The github page has information about how to develop it.